code/modules/mob/dead/observer/observer.dm

```diff
@@ -846,7 +846,7 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 /mob/dead/observer/is_literate()
 	return TRUE
 
-/mob/dead/observer/can_read(obj/O, check_for_light = FALSE)
+/mob/dead/observer/can_read(atom/viewed_atom, check_for_light = FALSE, check_literacy = TRUE, chat_messages = TRUE)
```

Comment:
```suggestion
/mob/dead/observer/can_read(atom/viewed_atom, check_for_light = FALSE)
```

So we don't actually need these args for what you are trying to do.  

Also I would recommend to use `silent` as an arg when you need to turn off `to_chat` messages since it's used in many places. (pretty much standard)  

Last if you have multiple TRUE/FALSE arguments, it's generally a good idea to consider making them all into defines and use a bitflag like I did with #64124.  It scales better the more args you add and it's easier to read.